packaging/debian/changelog

```diff
@@ -1,3 +1,9 @@
+mixxx (2.3.2-1~bionic) bionic; urgency=medium
+
+  * Build of 2.3.2
+
+ -- RJ Skerry-Ryan <rryan@mixxx.org>  Sun, 30 Jan 2022 20:37:54 +0000
+
 mixxx (2.3.1-1~bionic) bionic; urgency=medium
 
   * Build of 2.3.1
```

Comment:
whats going on with the version offset here?

Reply:
The debian tools automatically add the change log entry for the current build version. 
I have copied over just the auto-generated log entry form the last release. Else we would have a gap after the future release.

Reply:
thank you for the explanation.